content/_releases/22/28.md

```diff
@@ -0,0 +1,122 @@
+
+ ```json
+ {
+     "data": {
+         "type": "tenant",
+         "attributes": {
+             "ingress_config": {
+                 "annotations": {
+                     "app": {
+                         "nginx/123": "123",
+                         "nginx/456": "456"
+                     },
+                     "api": {
+                         "nginx/123": "123",
+                         "nginx/456": "456"
+                     },
+                     "webhooks": {
+                         "nginx/mtls": "true",
+                         "nginx/456": "456"
+                     },
+                     "apidocs": {
+                         "nginx/mtls": "true",
+                         "nginx/456": "456"
+                     }
+                 }
+             }
+         }
+     }
+ }
+ ```
+
+ </details>
+
+> PLease visit our [API documentation]({{site.data.tenant.apiBaseUri}}/docs/v2/#tenants) for more information.
+
+### Add "--push-retry" option to kaniko.
+
+ To improve push behaviour we added `--push-retry` flag in gitreceiver and made it configurable in gitreceiver only (default: 3). Please setup using `.Values.global.gitReceiver.dockerImageBuildJobPushRetry`.
+
+## OEM Related
+
+Information in this section is intended for our customers who use
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+### Support new component builder for java components
+
+We are introducing a new `сomponent-build-helper` [version](https://www.npmjs.com/package/@elastic.io/component-build-helper) `0.1.0`. It also means that the old appbuilder is totally deprecated and is no longer supported. We have added custom dockerfile support for Java components.
+
+> **Please Note:** For now our new component builder supports Java 8 based components only.
+
+### Check for vulnerabilities after build using grype
+
+Component vulnerabilities check in the build docker image is done by `сomponent-build-helper` during component build process. Components-build-helper has `build_component_docker` command which checks docker image vulnerabilities using grype tool.
+
+> For more information please visit our [article](/developers/component-build-configuration) on Components build configuration requirements.
+
+{% endif %}
+
+## Fixed bugs
+
+* Fixed bug where status page is not exposed at React pages if any maintenance is active. We added property "location" ("head" or "body", default: "head") into `tenant.attributes.customScripts`.
+
+> PLease visit our [API documentation]({{site.data.tenant.apiBaseUri}}/docs/v2/#tenants) for more information.
+
+## Components
+
+### [Stripe Component](/components/stripe/) `1.0.0`
+
+*   **ADDED** `Make Raw Request` Action
+
+### [Shopify admin component](/components/shopify) `1.6.3`
+
+*   **UPDATED** `Lookup Objects` action, now method `search` used instead of `list` for `customer` object type
```

Comment:
.., now **the** method 'search' **is** used instead of 'list' for **the** 'customer' object type.